### DISCUSSION TITLE
Bold Formatting Shortcut Issue (Command + B) After Switching from Windows to Mac in Freeplane

### DISCUSSION THREAD

**Discussion by BPA88** [freeplane/freeplane#1682] [Help]
Hello Freeplane Community,

I recently transitioned from using Freeplane on Windows to a Mac, and I've encountered an issue that I'm hoping to find some help with. On Windows, when I selected a portion of text within a node and pressed Ctrl + B, only the selected text would be bolded. However, now that I've switched to using Freeplane on my Mac, I've noticed that using the equivalent shortcut Command + B results in the entire node being bolded, not just the selected text.

Does anyone have any advice on how to restore the functionality so that Command + B only applies bold to the selected text, as it did on Windows? I would greatly appreciate any insights or suggestions on how to adjust the settings or fix this issue.

Thank you very much for your assistance and for sharing any experiences or solutions you might have!

_By the way, I want to express my gratitude to the creators and maintainers of Freeplane. This software has revolutionized my way of working, and I am immensely thankful for the impact it has had on my productivity and organizational skills._

**Comment by i-plasm** [comment]
Hi @BPA88 I can confirm what you describe occurs in macs. The good news is that you are able to use `Ctrl + B` (just as in Windows) in order to apply bold style to your selected text portion inside the node core. You can do the same for any other available styling: `Ctrl` + `U`/`I`... for underlined/italic/etc. You can get the full list of inline editor shortcuts by right clicking on your selected text portion and selecting the last submenu called `Format`. Hope it helps.

**Reply by BPA88** [reply]
Ohhh, that's perfect, thank you so much !!
